frontend/ui/widget/scrollhtmlwidget.lua

```diff
@@ -0,0 +1,150 @@
+--[[--
+HTML widget with vertical scroll bar.
+--]]
+
+local Device = require("device")
+local HtmlBoxWidget = require("ui/widget/htmlboxwidget")
+local Geom = require("ui/geometry")
+local GestureRange = require("ui/gesturerange")
+local HorizontalGroup = require("ui/widget/horizontalgroup")
+local HorizontalSpan = require("ui/widget/horizontalspan")
+local InputContainer = require("ui/widget/container/inputcontainer")
+local UIManager = require("ui/uimanager")
+local VerticalScrollBar = require("ui/widget/verticalscrollbar")
+
+local Input = Device.input
+local Screen = Device.screen
+
+local ScrollHtmlWidget = InputContainer:new{
+    html_body = nil,
+    css = nil,
+    default_font_size = 18,
+    htmlbox_widget = nil,
+    v_scroll_bar = nil,
+    dialog = nil,
+    dimen = nil,
+    width = 0,
+    height = 0,
+    scroll_bar_width = Screen:scaleBySize(6),
```

Comment:
Is this the same as in the text scroll widget?

Reply:
The text scroll widget specifies default size for width and height. It wasn't clear to me when would that be used so I left them as 0.

The scroll bar width and the text scroll pan size are the same.

```
    width = Screen:scaleBySize(400),
    height = Screen:scaleBySize(20),
    scroll_bar_width = Screen:scaleBySize(6),
    text_scroll_span = Screen:scaleBySize(12),
```